src/tracks/playabletrack/ui/PlayableTrackControls.cpp

```diff
@@ -158,6 +187,24 @@ void MuteAndSoloDrawFunction
       MuteOrSoloDrawFunction( dc, bev, pTrack, down, captured, true, hit );
    }
 }
+
+void EffectsDrawFunction
+( TrackPanelDrawingContext &context,
+  const wxRect &rect, const Track *pTrack )
+{
+   auto dc = &context.dc;
+
+   wxRect bev = rect;
+
+   GetEffectsHorizontalBounds( rect, bev );
+   {
+      auto target = dynamic_cast<EffectsButtonHandle*>( context.target.get() );
+      bool hit = target && target->GetTrack().get() == pTrack;
+      bool captured = hit && target->IsClicked();
+      bool down = captured && bev.Contains( context.lastState.GetPosition());
+      EffectsDrawFunction( dc, bev, pTrack, down, captured, hit );
```

Comment:
There are two things that isn't implemented in this PR: 
>Once an effect is applied, the ‘Add Effects’ button changes to a differently coloured ‘Effects’ button. This will help the user distinguish between tracks that have effects applied and tracks that don’t.

So text should change from 'Add Effect' to 'Effects'. and button color as well